firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -2387,6 +2386,13 @@ class BrowserViewController: UIViewController,
             // TODO: FXIOS-11248 Use NavigationBrowserAction instead of GeneralBrowserAction to open tab tray
             updateZoomPageBarVisibility(visible: false)
             focusOnTabSegment()
+            store.dispatch(
```

Comment:
Shouldn't we turn animations back on after the tab is displayed?

Reply:
We enable them when the `url` is changing, It should be enough in my opinion.

Reply:
Right. Unless the user selects the same tab again or hits "Done".